src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -539,10 +542,11 @@ internal bool ImportLibrary(string library)
                 return false;
             }
 
+            string path = library;
             if (!pathManager.ResolveLibraryPath(ref library))
             {
-                string errorMessage = string.Format(Properties.Resources.LibraryPathCannotBeFound, library);
-                OnLibraryLoadFailed(new LibraryLoadFailedEventArgs(library, errorMessage));
+                string errorMessage = string.Format(Properties.Resources.LibraryPathCannotBeFound, path);
+                OnLibraryLoadFailed(new LibraryLoadFailedEventArgs(path, errorMessage, !isLocalLib));
```

Comment:
Would you put a comment here linking `isLocalLib` to `throwOnFailure`, just found it hard to link these two variable by their meaning.

Reply:
Comment has been added